tests/testthat/test-epi_workflow.R

```diff
@@ -105,3 +105,40 @@ test_that("forecast method errors when workflow not fit", {
 
   expect_error(forecast(wf))
 })
+
+test_that("fit method does not silently drop the class", {
+  # This is issue #363
+
+  library(recipes)
+  tbl <- tibble::tibble(
+    geo_value = 1,
+    time_value = 1:100,
+    x = 1:100,
+    y = x + rnorm(100L)
+  )
+  edf <- as_epi_df(tbl)
+
+  rec_tbl <- recipe(y ~ x, data = tbl)
+  rec_edf <- recipe(y ~ x, data = edf)
+  expect_snapshot_warning(erec_tbl <- epi_recipe(y ~ x, data = tbl))
```

Comment:
suggest(non-blocking): error here like @brookslogan suggests in https://github.com/cmu-delphi/epipredict/issues/363 

main reason: while this might be convenient for some users, any downstream code will now have to do a boilerplate check whether the return is a recipe or an epi_recipe and implement the error itself, if it depends on it. seems better to just do that upstream.

Reply:
If you insist...